Lib/tempfile.py

```diff
@@ -242,26 +242,16 @@ def _mkstemp_inner(dir, pre, suf, flags, output_type):
     if output_type is bytes:
         names = map(_os.fsencode, names)
 
-    for seq in range(TMP_MAX):
```

Comment:
Why is this being removed? This block is supposed to continue trying to create the file until the maximum number of tries are reached ``TMP_MAX``.